core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -45,78 +45,14 @@
   // Crypto-related properties
   @Experimental
   CRYPTO_PREFIX("crypto.", null, PropertyType.PREFIX,
-      "Properties in this category related to the configuration of both default and custom crypto"
-          + " modules."),
+      "Properties related to encryption."),
```

Comment:
Brevity! But, perhaps a little bit more would help. These are specifically "data-at-rest" or "file-based" encryption.

Reply:
A little more for the description? Or you saying one of the properties should be "data-at-rest"?

Reply:
I'm saying 1) I like the shortening of the description, but 2) I think it should say something like "Properties related to on-disk file encryption" or "Properties related to encryption-at-rest" or whatever makes the most sense to clarify that it's separate from the RPC encryption properties.

Reply:
Roger